js/notifications.js

```diff
@@ -27,15 +27,23 @@
         },
         update: function() {
             const isFocused = window.isFocused();
+            const shouldPlayNotificationSound = Settings.isAudioNotificationSupported() &&
+                (storage.get('audio-notification') || false);
```

Comment:
Might be good to introduce an explaining variable for this line, given the `|| false`

Reply:
Good idea! Fixed it.